nox-support/doc-reqs.windows-amd64.lock.txt

```diff
@@ -10,8 +10,16 @@ sphinx==6.2.1 \
 sphinx-click==4.4.0 \
   --hash=sha256:2821c10a68fc9ee6ce7c92fad26540d8d8c8f45e6d7258f0e4fb7529ae8fab49 \
   --hash=sha256:cc67692bd28f482c7f01531c61b64e9d2f069bfcf3d24cbbb51d4a84a749fa48
+
+# N.B.: The 21c3f448d17224246e36a5db678e98b9261a1909604c160cd12f45fa1e7eb695 hash
+# was added manually. This is the hash of the 1.1.2 wheel which does not match the
+# VCS requirement this was locked from. The end result is docgen works on Windows
```

Comment:
Works minus displaying the upper-left logo when using `science doc open`. I think that classifies as NBD!